examples/get_cli_test_data.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import json
+import os
+
+import requests
+
+from starfish import Experiment
+from starfish.util.argparse import FsExistsType
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("experiment_url")
+    parser.add_argument("output_dir", type=FsExistsType())
+    args = parser.parse_args()
+
+    # save image stacks locally
+    exp = Experiment.from_json(args.experiment_url + 'experiment.json')
```

Comment:
Sorry I didn't notice this the first time; you should use `pathlib` to do these manipulations:

`str(pathlib.PurePosixPath(args.experiment_url, "experiment.json"))`

Reply:
What's the advantage of using pathlib to format the urls? When I do this the resulting url is https:///dmf0bdeheu4zf.cloudfront.net/20180919/DARTFISH-TEST/experiment.json which throws an invalid url error because of the extra /

Reply:
ok that's unfortunate.  pathlib has a very intriguing interface, but may not work the best for URLs.  `os.posixpath.join` will definitely work though.